bpf/pkt_drops.h

```diff
@@ -17,7 +17,10 @@ static inline int trace_pkt_drop(void *ctx, u8 state,
     u16 family = 0,flags = 0;
 
     id.if_index = skb->skb_iif;
-
+    // filter out TCP sockets with unknown or loopback interface
+    if (id.if_index == 0 || id.if_index == 1) {
```

Comment:
is it 100% sure that if_index 0 or 1 is always lo ?

Reply:
yes those two index are reserved , I was hopping if there is api or helper but couldn't find any and saw other app doing the same check to detect lo